### ISSUE TITLE
Hex byte password support

### ISSUE THREAD

**Issue by danielpclark** [danielpclark/abrute#30]
I've recently received a request to be able to enter raw `u8` byte values via hex escape codes as password entry.  I agree that this should be supported but allow me first to specify what's required to accomplish this.

Currently this program executes all password entries through **STDIN** from the command line command of `aescrypt` and `unzip`.  As this program has optimizations for cluster work and non-adjacent character algorithms the only passwords this currently supports is **UTF8** characters.

So for entering hex byte codes for an aescrypt password there are two ideal solutions that need to be implemented.

 1. The [digits](https://github.com/danielpclark/digits) library needs to be overhauled to support non UTF8 characters.  The _digits_ library handles all the math involved for character sequences involving server clusters and non-adjacent progressions.
 2. I believe it would be best to switch away from using **STDIN** and the command line application `aescrypt` and implement a native Rust library for handling AES files and iostreams.  This project was already started but is kind of on hold until I have time for it.  You can see it here at [aescry](https://github.com/danielpclark/aescry).  Aescry is largely a WIP translation of C source code based on `aescrypt`, the libraries `aescrypt` is based on, and the official AES spec.  As of this writing only the SHA256SUM feature has been finished.  Once this library is finished the performance for password attempts per second in Abrute should be significantly better.

So there's a lot of work that has yet to be done to make this possible.  I wouldn't advise implementing it in `abrute` any other way as this is no mere simple iterative script.
